commands/util.go

```diff
@@ -117,7 +118,13 @@ func driversForNodeGroup(ctx context.Context, dockerCli command.Cli, ng *store.N
 					}
 				}
 
-				d, err := driver.GetDriver(ctx, "buildx_buildkit_"+n.Name, f, dockerapi, imageopt.Auth, kcc, n.Flags, n.Files, n.DriverOpts, n.Platforms, contextPathHash)
+				buildkitAPI, err := buildkitclient.New(ctx, endpoint)
+				if err != nil {
+					di.Err = err
+					return nil
+				}
+
+				d, err := driver.GetDriver(ctx, "buildx_buildkit_"+n.Name, f, dockerapi, buildkitAPI, imageopt.Auth, kcc, n.Flags, n.Files, n.DriverOpts, n.Platforms, contextPathHash, endpoint)
```

Comment:
I think this functions needs some refactoring, the list of args is becoming very hard to read...